test/state/state.hpp

```diff
@@ -127,7 +127,7 @@ struct Transaction
 
     Type type = Type::legacy;
     bytes data;
-    int64_t gas_limit;
+    int64_t gas_limit{};
```

Comment:
```suggestion
    int64_t gas_limit = 0;
```
to be consistent with initializations below

Reply:
done